Source/WebCore/Modules/url-pattern/URLPatternParser.cpp

```diff
@@ -310,7 +310,7 @@ String generateSegmentWildcardRegexp(const URLPatternStringOptions& options)
 template<typename CharacterType>
 static String escapeRegexStringForCharacters(std::span<const CharacterType> characters)
 {
-    static constexpr std::array regexEscapeCharacters { '.', '+', '*', '?', '^', '$', '{', '}', '(', ')', '[', ']', '|', '/', '\\' }; // NOLINT
+    static constinit std::array<CharacterType, 15> regexEscapeCharacters { '.', '+', '*', '?', '^', '$', '{', '}', '(', ')', '[', ']', '|', '/', '\\' }; // NOLINT
```

Comment:
This should be changed back from `constinit`, which I misunderstood as better than `constexpr`:
```suggestion
    static constexpr std::array<CharacterType, 15> regexEscapeCharacters { '.', '+', '*', '?', '^', '$', '{', '}', '(', ')', '[', ']', '|', '/', '\\' }; // NOLINT
```